docs/content/users/extend/custom-commands.md

```diff
@@ -30,6 +30,22 @@ To provide host commands, place a Bash script in `.ddev/commands/host`. For exam
 open -a PhpStorm.app ${DDEV_APPROOT}
 ```
 
+## Web Commands
```

Comment:
I'm confused why "Container commands" that is already there doesn't add enough information for you. Why are we adding a separate "Web commands" section? And if we're going to add it, shouldn't it go below the "Container commands" section, which should have the same info?

Reply:
I added them, since you suggested this:
> You'd also want to add a mention of this in users/extend/custom-commands.md

The changes to _users/extend/custom-commands.md_ can be removed from this PR, or slimmed down?